config.yaml

```diff
@@ -178,6 +178,24 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-1.9.3.4
+        version: 1.9.3.4
+        dist:
+          url: https://github.com/OpenMage/magento-mirror/archive/1.9.3.4.zip
+          type: zip
+          shasum: 6dadf8de337ae7033839839caeb15cbb2e98e4b8
+        extra:
+          sample-data: sample-data-1.9.2.4
+
+      - name: magento-ce-1.9.3.3
```

Comment:
This name is reserved for original Magento CE packages by the vendor, not OpenMage as used in this entry.